angularSTEP/functions/src/index.ts

```diff
@@ -1,8 +1,28 @@
 import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin'
+import algoliasearch from 'algoliasearch'
 
-// // Start writing Firebase Functions
-// // https://firebase.google.com/docs/functions/typescript
-//
-// export const helloWorld = functions.https.onRequest((request, response) => {
-//  response.send("Hello from Firebase!");
-// });
+admin.initializeApp();
+const env = functions.config();
+
+const client = algoliasearch(env.algolia.appid, env.algolia.apikey);
```

Comment:
You need to sent your own appid and apikey config variables to your local computer.
`cd` into the projects `functions` directory put `firebase functions:config:set algolia.appid='{{put the api key in here}}' algolia apikey='{put the api key in here}'` into the terminal